backend/mailman_client/mailman.py

```diff
@@ -0,0 +1,25 @@
+from dropbox import Dropbox, files
+import string
+
+MAILMAN_CLIENT_ID = '2j1mfqvrkfz4n65'  # not the best way to store this but for testing/simplicity for now
+
+class MailmanClient():
+    def __init__(self, access_token):
+        self.dbx = Dropbox(
+            oauth2_access_token=access_token,
```

Comment:
Since our logic will likely be a single client uploading for a lot of different users, can we create a client with just the client_id and then pass in a token to upload file?

Reply:
Yea that makes sense to me.

Reply:
Oh wait actually, I think we should instantiate the Dropbox obj with the token so we can refresh it otherwise, after 4 hours the token would be useless